Gemfile

```diff
@@ -3,7 +3,7 @@
 
 source 'https://rubygems.org'
 
-gem 'puppet', ENV.key?('PUPPET_VERSION') ? "~> #{ENV['PUPPET_VERSION']}" : '>= 4.6'
+gem 'puppet', ENV.key?('PUPPET_VERSION') ? "~> #{ENV['PUPPET_VERSION']}" : '>= 5.5'
```

Comment:
Should this be 5.5.8 to match the metadata?

Reply:
This comes from our modulesync config. Given in our travis config we explicitly set the version and the chance we'll install 5.5.0 is minimal I don't care that much about it.